changelog.d/1699

```diff
@@ -0,0 +1,7 @@
+synopsis: Add NamedRoutes instance to IsElem
+prs: #1699
+issues: #1674
+description: {
+Add missing IsElem instance for NamedRoutes, this allows links to be checked
+`safeLink`. 
```

Comment:
Is this a stray line?

Reply:
Just wrapped at that point, though I realised it was missing a 'with'.